src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -125,4 +127,36 @@ export class AppController {
             },
         );
     }
+
+    public getServerLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(
+                    `drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`,
+                ),
+            )
+            .toString();
+    }
+
+    public getAdbLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(`drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/adb.log`),
```

Comment:
We build the path to adb.log in 3 different places. Would it improve the cohesion if we pulled that construction into a single method that we referenced from the other locations?

Reply:
Yes! I also moved this logic out of `AppController` and into a new `LogController` class